airflow/dags/create_external_tables/ntd_data_products/annual_database_agency_information.yml

```diff
@@ -0,0 +1,139 @@
+operator: operators.ExternalTable
+bucket: gs://calitp-ntd-data-products
+prefix_bucket: true
+post_hook: |
```

Comment:
Do we want to keep using these (post-hooks)?

Reply:
I'm ambivalent, I think they're useful quick tests on small data sets